examples/cull-idle/cull_idle_servers.py

```diff
@@ -312,6 +320,11 @@ def handle_user(user):
            help="""Cull users in addition to servers.
                 This is for use in temporary-user cases such as tmpnb.""",
            )
+    define('user_timeout', default=0, type=int,
+           help="""Inactive time for culling users.  Defaults to --timeout.""",
+           )
+    define('user_max_age', default=0,
+           help="The maximum age (in seconds) of users that should be culled even if they are active.  Defaults to --max-age.")
```

Comment:
Perhaps: The age limit (in seconds) where users, active and inactive, will be culled if user's age exceeds the limit. Defaults to --max_age